# Define msg for odom ratio
float32 odom_ratio